libr/core/cmd_type.c

```diff
@@ -745,6 +746,66 @@ static int cmd_type(void *data, const char *input) {
 				r_core_cmd_help (core, help_msg_ts);
 			}
 			break;
+		case 'c': { //tsc
+			char *name = NULL;
+			size_t varname_len = 0;
+			int prev_size = 0;
+			char *varname = NULL;
+           		SdbKv *kv;
+           		SdbListIter *iter;
+			int i, j;
+        		SdbList *l = sdb_foreach_list (TDB, true);
+        		ls_foreach (l, iter, kv) {
+        			if (!strcmp (sdbkv_value (kv), "struct")) {
+                			if (!name || strcmp (sdbkv_value (kv), name)) {
+                				free (name);
+                    				name = strdup (sdbkv_key (kv));
+                    				const char *q = sdb_fmt ("struct.%s", name);
+                        			const char *res = sdb_get (TDB, q, 0);
```

Comment:
Hey hi , as @radare said can u use sdb_const_get(TDB, ...) , since it saves an allocation and returns const char *